docs/usage/tutorial.md

```diff
@@ -0,0 +1,113 @@
+```
+MATCH (a:AWSAccount)-[:RESOURCE]->(rds:RDSInstance{storage_encrypted:false})
+RETURN a.name, rds.id
+```
+
+![Unencrypted RDS instances](../images/unencryptedinstances.png)
+
+
+The results show up in a table because we specified attributes like `a.name` and `rds.id` in our return statement (as opposed to having it `return *`).  We used the "{}" notation to have the query only return RDSInstances where `storage_encrypted` is set to `False`.
+
+If you want to go back to viewing the graph and not a table, simply make sure you don't have any attributes in your return statement -- use `return *` to return all nodes decorated with a variable label in your `MATCH` statement, or just return the specific nodes and relationships that you want.
+
+Let's look at some other AWS assets now.
+
+
+### Which [EC2](https://aws.amazon.com/ec2/) instances are directly exposed to the internet?
+```
+MATCH (instance:EC2Instance{exposed_internet: true})
+RETURN instance.instanceid, instance.publicdnsname
+```
+![EC2 instances open to the internet](../images/ec2-inet-open.png)
+
+These instances are open to the internet either through permissive inbound IP permissions defined on their EC2SecurityGroups or their NetworkInterfaces.
+
+If you know a lot about AWS, you may have noticed that EC2 instances [don't actually have an exposed_internet field](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Instance.html).  We're able to query for this because Cartography performs some [data enrichment](#data-enrichment) to add this field to EC2Instance nodes.
+
+
+### Which [S3](https://aws.amazon.com/s3/) buckets have a policy granting any level of anonymous access to the bucket?
+```
+MATCH (s:S3Bucket)
+WHERE s.anonymous_access = true
+RETURN s
+```
+![S3 buckets that allow anon access](../images/anonbuckets.png)
+
+These S3 buckets allow for any user to read data from them anonymously.  Similar to the EC2 instance example above, S3 buckets returned by the S3 API [don't actually have an anonymous_access field](https://docs.aws.amazon.com/AmazonS3/latest/API/API_Bucket.html) and this field is added by one of Cartography's [data augmentation steps](#data-augmentation).
+
+A couple of other things to notice: instead of using the "{}" notation to filter for anonymous buckets, we can use SQL-style `WHERE` clauses.  Also, we used the SQL-style `AS` operator to relabel our output header rows.
+
+
+### How many unencrypted RDS instances do I have in all my AWS accounts?
+
+Let's go back to analyzing RDS instances.  In an earlier example we queried for RDS instances that have encryption turned off.  We can aggregate this data by AWSAccount with a small change:
+
+```
+MATCH (a:AWSAccount)-[:RESOURCE]->(rds:RDSInstance)
+WHERE rds.storage_encrypted = false
+RETURN a.name as AWSAccount, count(rds) as UnencryptedInstances
+```
+![Table of unencrypted RDS instances by AWS account](../images/unencryptedcounts.png)
+
```

Comment:
Thanks for pointing this out, I think I'll actually put the link here: https://github.com/lyft/cartography/blob/1ebb31458805a143bed2ea7adbe9ccc297106f7d/docs/usage/tutorial.md#%E2%84%B9%EF%B8%8F-already-know-how-to-query-neo4j--you-can-skip-to-our-reference-material

Which I will fix because these are broken links.